index.js

```diff
@@ -1,39 +1,36 @@
 module.exports = { isValid };
 
+/**
+ * Module dependencies.
+ */
+
+var is = require('is');
+
+
 /**
  * Validate phone number
  * @param  {string}  n
- * @return {Boolean} 
+ * @return {Boolean}
  */
-function isValid(n) {
-
-  var firstChar;
+function isValid(n, callback) {
+  if ('function' != typeof callback) callback = function () {};
```

Comment:
This is currently not backward compatible & will break existing code.

I'd check if `callback` argument was passed in the first place & use promises then.

Reply:
Better still we update the `package.json` version to `2.0.2`, stating that it's not backward compatible, or add a CHANGELOG.md showing the changes.

Reply:
Btw, `"I'd check if callback argument was passed in the first place & use promises then."`, this was implemented 

```
16 | if ('function' != typeof callback) callback = function () {};
34 | promise.then(callback.bind(null, null), callback);
``` 
resolving with either a promise or callback